frontend/src/components/common/form/AutoCompleteText.tsx

```diff
@@ -75,6 +75,8 @@ export const AutoCompleteText: React.FC<IAutoCompleteProps> = ({
       } else {
         dynamicSuggestions = options.filter(v => regex.test(v.label));
       }
+    } else {
+      setFieldValue(field, null);
```

Comment:
Does formik throw an error in the console for this? If so you may need to set this to '' instead.

Reply:
It did nt throw an error, but I think your suggestion is safer